223/borisov

rdydryliu opened the pull request:
Контрольные вопросы:
[11ИС-223Д Борисов Иван. Контрольные воппросы к лабораторной 6.docx](https://github.com/user-attachments/files/17699182/11.-223.6.docx)

rdydryliu committed:
Текущая ветка: 223/borisov
	новый файл:    223/borisov/arithmetic.sh для вычислений
	новый файл:    223/borisov/avg.sh для вычилления среднего числа
	новый файл:    223/borisov/homedir.sh для поиска домашней директории
	новый файл:    223/borisov/touchdir.sh для создания директории

Lyncore commented on 223/borisov/avg.sh:
```diff
@@ -0,0 +1,24 @@
+echo "Введите второе число"
+read b
+echo "Введите третье число"
+read c
```
нужно реализовать ввод произвольного количества чисел путем передачи их в качестве аргумента

Lyncore commented on 223/borisov/homedir.sh:
```diff
@@ -0,0 +1,11 @@
+username="doc"
+h=""
+h=`grep $username /etc/passwd`
+echo $h
```
нужно реализовать выборку адреса домашней директории из целой строки

Lyncore commented on 223/borisov/touchdir.sh:
```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+dir="/home/evan/test/tes/test"
```
нужно принимать название директории из аргументов, переданных при запуске скрипта

Lyncore commented on 223/borisov/homedir.sh:
```diff
@@ -0,0 +1,11 @@
+
+#!/bin/bash
+username="doc"
```
нужно принимать имя пользователя из аргументов, переданных при запуске скрипта

Lyncore commented on 223/borisov/arithmetic.sh:
```diff
@@ -0,0 +1,19 @@
+echo "Введите  операцию"
+read c
+echo "Введите второе число"
+read b
```
нужно принимать числа из аргументов, переданных при запуске скрипта

rdydryliu committed:
изменено:      arithmetic.sh числа вводятся при запуске скрипта
	изменено:      avg.sh числа вводятся при запуске скрипта
	изменено:      homedir.sh Пользователь вводятся при запуске скрипта
	изменено:      touchdir.sh Дирректория вводятся при запуске скрипта
